src/basic/calendarspec.c

```diff
@@ -876,6 +884,53 @@ static int parse_calendar_time(const char **p, CalendarSpec *c) {
         return r;
 }
 
+static int is_valid_timezone(char *tz, bool *result)
```

Comment:
there's already timezone_is_valid() defined in src/basic/time-util.c. Please use that.

Reply:
@poettering yep, it was removed 2 days ago, not sure how you ended up commenting the obsolete commit. Apologise for not squashing it first.

Reply:
Ah, please always push "perfect" PRs. I.e. push PRs that do not reflect your development history but rather reflect logical steps, that are easy to review. Or to say this differently: we want each commit to be nicely bisectable.